pkg/services/sqlstore/migrator/column.go

```diff
@@ -22,3 +29,33 @@ func (col *Column) String(d Dialect) string {
 func (col *Column) StringNoPk(d Dialect) string {
 	return d.ColStringNoPk(col)
 }
+
+// Validate will return an error if this Column has any validation issues.
+// This currently errors if a DB_Bool type column has an invalid Default literal.
+func (col *Column) Validate() error {
+	if col.Type == DB_Bool {
+		if _, err := ParseBoolStr(col.Default); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// ParseBoolStr parses the string representation of the default value in a bool column.
+// This is meant to ensure that all dialects support the same literals in the same way.
+func ParseBoolStr(s string) (*bool, error) {
+	switch s {
+	case "1", "t", "T", "true", "TRUE", "True":
```

Comment:
If we want to support all the same inputs as Go, should we replace this switch with `strconv.ParseBool`?